_sphinx/conf.py

```diff
@@ -66,6 +67,21 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = False
 
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+extensions = ['sphinx.ext.intersphinx']
+
+# Intersphinx mapping
+
+intersphinx_mapping = {
+    'catkin_pkg':    ('http://docs.ros.org/independent/api/catkin_pkg/html', None),
+    'jenkins_tools': ('http://docs.ros.org/independent/api/jenkins_tools/html', None),
+    'rosdep':        ('http://docs.ros.org/independent/api/rosdep/html', None),
+    'rosdistro':     ('http://docs.ros.org/independent/api/rosdistro/html', None),
+    'rosinstall':    ('http://docs.ros.org/independent/api/rosinstall/html', None),
+    'rospkg':        ('http://docs.ros.org/independent/api/rospkg/html', None),
+    'vcstools':      ('http://docs.ros.org/independent/api/vcstools/html', None)
```

Comment:
If I understood right, this doens't look like it will scale. Will we need to add a line for every single package we want to link to?

Reply:
Hmm, that's a good point. But I think that isn't something we can't really deal with here. It depends on how much ground each Sphinx build covers. If we do so on per package basis, then yes, we have a scalability problem unless we somehow aggregate Sphinx inventory files. On the other hand, if we do such builds on a per distro basis (akin to how it is done for Python versions => https://docs.python.org/3/objects.inv), then we're fine.

@tfoote @dirk-thomas thoughts?

Reply:
I think requiring a line for every "external" package is fine - I don't see another way. What we shouldn't require going forward is entries for all ROS packages for which documentation is being generated, e.g. `rclcpp`.

Reply:
For those, were you think of some automatic discovery mechanism for .inv files?

Reply:
Since we have control about generating them we should make sure that are used "automatically" here.